projects/core/src/cart/store/actions/cart-entry.action.ts

```diff
@@ -36,13 +36,13 @@ export class CartAddEntrySuccess extends EntityProcessesDecrementAction {
     public payload: {
       userId: string;
       cartId: string;
-      productCode: string;
-      quantity: number;
-      deliveryModeChanged: boolean;
-      entry: OrderEntry;
-      quantityAdded: number;
-      statusCode: string;
-      statusMessage: string;
+      productCode?: string;
+      quantity?: number;
```

Comment:
I think those 2 can remain mandatory, because we take them from the original AddToCart action (not from the response from backend). Is it right?

Same for the `CartAddEntrySuccessEvent`

Reply:
You are right. Didn't notice for qty first